oem2orm/oep_oedialect_oem2orm.py

```diff
@@ -373,18 +375,26 @@ def api_updateMdOnTable(metadata, token=None):
     schema = getTableSchemaNameFromOEM(metadata)[0]
     table = getTableSchemaNameFromOEM(metadata)[1]
 
-    logging.info("UPDATE METADATA")
+    logging.info(f"Update metadata: {table}")
     api_action = setupApiAction(schema, table, token)
     resp = requests.post(api_action.dest_url, json=metadata, headers=api_action.headers)
     if resp.status_code == 200:
-        logging.info("   ok.")
-        logging.info(api_action.dest_url)
+        logging.info(f"METADATA SUCCESSFULLY UPDATED: {table}")
+        logging.info(f"Link to updated metadata on OEP: {api_action.dest_url}")
     else:
-        error_msg = resp.json()
-        logging.info(error_msg)
-        logging.info("HTTP status code: ")
-        logging.info(resp.status_code)
-        raise MetadataError(f"Uploading of metadata failed. Response from OEP: {error_msg}")
+        oep_err_msg_header = re.search("<h3>(.*)</h3>", resp.text).group(1)
```

Comment:
i think you can avoid the re. search by using resp.reason ... but not sure :)

Reply:
I think I'll leave it as is for now. :)